Sprint-2/interpret/invert.js

```diff
@@ -6,24 +6,42 @@
 
 // E.g. invert({x : 10, y : 20}), target output: {"10": "x", "20": "y"}
 
-function invert(obj) {
-  const invertedObj = {};
+// function invert(obj) {
+//   const invertedObj = {};
 
-  for (const [key, value] of Object.entries(obj)) {
-    invertedObj.key = value;
-  }
+//   for (const [key, value] of Object.entries(obj)) {
+//     invertedObj.key = value;
+//   }
 
-  return invertedObj;
-}
+//   return invertedObj;
+// }
 
 // a) What is the current return value when invert is called with { a : 1 }
+// 1 
 
 // b) What is the current return value when invert is called with { a: 1, b: 2 }
+///2
```

Comment:
I don't think the function currently returns numbers. Can you talk through why you think it does?